README.md

```diff
@@ -63,3 +63,95 @@ Most of the translations can be found in the web client but we have several othe
+
+## Server Development
+
+These instructions will help you get set up with a local development environment in order to contribute to this repository. Before you start, please be sure to completely read our [guidelines on development contributions](https://jellyfin.org/docs/general/contributing/development.html). Note that this project is supported on all major operating systems (Windows, Mac and Linux).
+
+### Prerequisites
+
+Before the project can be built, you must first install the [.NET Core 3.1 SDK](https://dotnet.microsoft.com/download) on your system.
+
+Instructions to run this project from the command line are included here, but you will also need to install an IDE if you want to debug the server while it is running. Any IDE that supports .NET Core development will work, but explicit instructions for [Visual Studio](https://visualstudio.microsoft.com/downloads/) (at least 2017) and [Visual Studio Code](https://code.visualstudio.com/Download) are included here.
+
+### Cloning the Repository
+
+After dependencies are installed you will need to clone a local copy of this repository. If you just want to run the server from source you can clone this repository directly, but if you are intending to contribute code changes to the project, you should [set up your own fork](https://jellyfin.org/docs/general/contributing/development.html#set-up-your-copy-of-the-repo) of the repository. The following example shows how you can clone the repository directly over HTTPS.
+
+```bash
+git clone https://github.com/jellyfin/jellyfin.git
+```
+
+### Installing the Web Client
+
+By default, the server is configured to host the static files required for the [web client](https://github.com/jellyfin/jellyfin-web) in addition to serving the backend API. before you can run the server, you will need to get a copy of the web client files since they are not included in this repository directly.
+
+Note that it is also possible to [host the web client separately](#hosting-the-web-client-separately) from the web server with some additional configuration, in which case you can skip this step.
+
+There are two options to get the files for the web client:
+
+1. Build them from source following the instructions on the [jellyfin-web repository](https://github.com/jellyfin/jellyfin-web)
+2. Get the pre-built files from an existing installation of the server. For example, with a Windows server installation the client files are located here: `C:\Program Files\Jellyfin\Server\jellyfin-web`
+
+Once you have a copy of the built web client files, you need to copy them into the build output directory of the web server project. For example: `<repository root>\Jellyfin.Server\bin\Debug\netcoreapp3.1\jellyfin-web`
```

Comment:
```suggestion
Once you have a copy of the built web client files, you need to copy them into the build output directory of the web server project.

* `<repository root>/jellyfin-web`
* `<repository root>/Jellyfin.Server/bin/Debug/netcoreapp3.1/jellyfin-web`
```

Reply:
Not sure I understand this change. What is the first bullet point line `<repository root>/jellyfin-web` supposed to be?

Reply:
You can also add the source there instead and it will build fine.

Reply:
I didn't know that was an option XD

The path to use is actually `<repository root>/Mediabrowser.WebDashboard/jellyfin-web`. 

I've also removed my original path for the build output from the doc here since that is specific to the build configuration (Debug, Release, framework, etc) and I can't see why using that would be preferable to the more flexible option.